Source/WebCore/css/parser/CSSTokenizer.h

```diff
@@ -131,4 +132,7 @@ class CSSTokenizer {
     CSSTokenizerInputStream m_input;
 };
 
+bool isWhitespace(CSSParserTokenType);
```

Comment:
We could make those `static` inside `CSSTokenizer` to prevent WebCore "pollution" with a pure CSS (weird) concept.

Reply:
Made both isNewline and isWhitespace static methods inside of `CSSTokenizer`.